test/blackbox-tests/test-cases/pkg/opam-generate-ocaml-package.t

```diff
@@ -18,3 +18,11 @@
   (repositories
    (complete false)
    (used))
+  
+  (solver_env
+   (flags
+    ("with-doc" "with-test"))
+   (sys)
```

Comment:
If it's empty, we might as well omit it.

Reply:
I thought about removing empty sections but figured it would be useful to tell users explicitly that no system variables were set. Otherwise in such a case they might think that system variables aren't supposed to show up here.